index.Rmd

```diff
@@ -57,7 +57,7 @@ map_1 <- tm_basemap(leaflet::providers$Esri.WorldTopoMap, alpha = 0.4) +
 map_1
```

Comment:
Could you include a legend in map 1? For example, what are the green dots and yellow dots?